template.yaml

```diff
@@ -35,7 +35,7 @@ Resources:
             TableName: '*'
       Environment: # More info about Env Vars: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#environment-object
         Variables:
-          INDEX_BUCKET: acl-lucene-index
+          INDEX_BUCKET: adamyy-anlessini
```

Comment:
We might want to parameterize this in a further PR. I will revert my change here.

The user will need to create their own bucket and populate the bucket with index files. We might want to provide instructions on how to.